babeld/babel_interface.c

```diff
@@ -323,6 +323,23 @@ DEFUN (babel_set_wired,
     return CMD_SUCCESS;
 }
 
+DEFUN (no_babel_set_wired,
```

Comment:
Please combine what is possible into the same commands. For example `no_babel_set_wired_cmd` with `babel_set_wired_cmd`, etc. And we require to use `DEFPY` instead of `DEFUN`. We need to switch to this.

Using DEFPY requires fewer changes (mostly) than duplicating the blocks for negative commands.

Reply:
Updated. Could you have another look?